scripts/common/sonic-telemetry-build/build.sh

```diff
@@ -1,31 +1,11 @@
 #!/bin/bash -ex
 
+# Build script for sonic-telemetry.
+# Assumes sonic-mgmt-common is already cloned & built.
+
 # Install HIREDIS
 sudo dpkg -i buildimage/target/debs/stretch/libhiredis*.deb
 
-# Install REDIS
-sudo apt-get install -y liblua5.1-0 lua-bitop lua-cjson
```

Comment:
check out this one.

https://github.com/Azure/sonic-build-tools/commit/db20bdbdd46efdabf32dce6a1a62baa0a1e96483

We should still keep the redis-server

Reply:
Hi Guohan, thanks for quick review. Redis installation and startup is handled in sonic-mgmt-common build script -- https://github.com/Azure/sonic-build-tools/blob/master/scripts/common/sonic-mgmt-common-build/build.sh#L10.
Telemetry jenkinsfile will invoke that script directly -- https://github.com/Azure/sonic-build-tools/pull/129/files#diff-30462f129a50d9d0b41b9609d08f147dR24.

Redis, libyang etc are common for mgmt-common, mgmt-framework and telemetry builds. More will come as sonic-mgmt-common's code evolves in future. Hence keeping all sonic-mgmt-common specific dependencies in one place.. Telemetry build.sh will install the dependencies that are specific to telemetry code -- like hiredis here.